package.json

```diff
@@ -45,7 +45,7 @@
     "husky": "~4.2.5",
     "lint-staged": "~10.2.11",
     "mocha": "8.0.1",
-    "mongodb": "3.5.9",
+    "mongodb": "3.6.4",
```

Comment:
Should be noted in CHANGES_NEXT_RELEASE, this way:

```
- Upgrade mongodb dev dependency from 3.5.9 to 3.6.4 (#204)
```

(I'm not sure about citing #204 in this case...)

Reply:
Fixed 31dfddf